components/developer/fmtlib/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2023 Friedrich Kink
+#
+
+BUILD_BITS=		64
+BUILD_STYLE=		cmake
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=         fmt
+COMPONENT_VERSION=      9.1.0
+HUMAN_VERSION=          $(COMPONENT_VERSION)
+COMPONENT_SUMMARY=      {fmt} is an open-source formatting library providing a fast and safe alternative to C stdio and C++ iostreams.
+COMPONENT_PROJECT_URL=  https://github.com/fmtlib/$(COMPONENT_NAME)
```

Comment:
What is the purpose of using [fmt} in the summary?

Reply:
there is no special purpose. it is just copy paste from the project. Should I remove?

Reply:
Yes please, it looks odd.